src/processing/resources/manifest.js

```diff
@@ -2,7 +2,7 @@ import ResourceProcessorBase from './resource-processor-base';
 import trim from '../../utils/string-trim';
 
 class ManifestProcessor extends ResourceProcessorBase {
-    processResource (manifest, ctx, charset, urlReplacer) {
+    processResource (manifest, ctx, contentCharset, urlReplacer) {
```

Comment:
Here and in all other places: using `charset` instead of `contentCharset` is just fine